### PR TITLE
Add Issue Mapper/Model/Controller/ControllerTest

### PR DESCRIPTION
**Pull request by Yarik2308**
https://github.com/malinink/easy-tests/issues/327

- [x] controller
- [x] mapper
- [x] model
- [x] test
- [x] remove list realization

### CODE REVIEW

File: src/main/java/easytests/api/v1/models/Issue.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+/**
+ * @author Yarik2308
+ */
+@Data
+public class Issue {
+    private Integer id;
+
+    private String name;
+
+    private Integer subjectId;
```
Comment (malinink) [inline src/main/java/easytests/api/v1/models/Issue.java]: subject

File: src/main/java/easytests/api/v1/controllers/IssuesController.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    protected IssuesService issuesService;
+
+    @Autowired
+    protected IssuesOptionsBuilder issuesOptions;
+
+    @Autowired
+    @Qualifier("IssuesMapperV1")
+    private IssuesMapper issuesMapper;
+
+    /**
```
Comment (malinink) [inline src/main/java/easytests/api/v1/controllers/IssuesController.java:28]: let's use short notation from API doc. i.e. list, create, etc.

File: src/test/java/easytests/api/v1/controllers/IssuesControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package easytests.api.v1.controllers;
+
+/**
+ * @author Yarik2308
+ */
+public class IssuesControllerTest {
+    /**
```
Comment (malinink) [inline src/test/java/easytests/api/v1/controllers/IssuesControllerTest.java]: you don't need to determine all tests here, just add separators on each method. (so it must equals to controller comments)

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by Yarik2308** [comment]
@malinink Done.

File: src/main/java/easytests/api/v1/models/Issue.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+/**
+ * @author Yarik2308
+ */
+@Data
+public class Issue {
+    private Integer id;
+
+    private String name;
+
+    private Integer subject;
```
Comment (malinink) [inline src/main/java/easytests/api/v1/models/Issue.java]: By documentation it is not an integer.

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by Yarik2308** [comment]
@malinink fixed.

**Comment by malinink** [comment]
@Yarik2308 checkstyle failed, fix.

**Comment by Yarik2308** [comment]
@malinink Done

**Comment by malinink** [comment]
@Yarik2308 looks good!

**Review by malinink — APPROVED** [review]

**Pull request merged**